modules/installation-requirements-user-infra.adoc

```diff
@@ -41,10 +46,27 @@ operating system.
 [id="network-connectivity_{context}"]
 == Network connectivity requirements
 
+ifeval::["{context}" == "installing-ibm-z"]
```

Comment:
You could make more of this statement common.

Reply:
Done, totally missed this, thanks :)